src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -607,6 +607,10 @@ export function selectAtlasConfirmationCode(appointment) {
   return appointment?.videoData?.atlasConfirmationCode;
 }
 
+export function selectAppointmentTravelClaim(appointment) {
+  return appointment?.vaos?.apiData?.claim;
```

Comment:
I'm assuming this will ultimately depend on BE implementation which hasn't been finalized yet?

Reply:
Yes this will likely change once the backend if finalized.